docs/Developer Resources/Building ZFS.rst

```diff
@@ -5,24 +5,24 @@ GitHub Repositories
 ~~~~~~~~~~~~~~~~~~~
 
 The official source for ZFS on Linux is maintained at GitHub by the
-`zfsonlinux <https://github.com/zfsonlinux/>`__ organization. The
-project consists of two primary git repositories named
-`spl <https://github.com/zfsonlinux/spl>`__ and
-`zfs <https://github.com/zfsonlinux/zfs>`__, both are required to build
-ZFS on Linux.
+`openzfs <https://github.com/openzfs/>`__ organization. The project
+consists of two primary git repositories named `spl
+<https://github.com/openzfs/spl>`__ and `zfs
+<https://github.com/openzfs/zfs>`__, both are required to build ZFS on
+Linux.
```

Comment:
The `spl` repo hasn't been used for quite a while now, I think you can remove references to it (per the note below), and just address building 0.8 and later code.

Reply:
@ahrens pushed a new commit removing all references to building <= 0.7. Sorry for the ugly diff :/